core/cmd/hoverfly/main.go

```diff
@@ -194,6 +206,28 @@ func main() {
 		})
 	}
 
+	if *logsOutput == "file" {
+		file, err := os.OpenFile(*logsFile, os.O_CREATE|os.O_WRONLY|os.O_APPEND, 0666)
+		if err == nil {
+			log.SetOutput(file)
+		} else {
+			log.Fatal("Failed to write log file:" + *logsFile)
+		}
+	} else {
+		if isFlagPassed("logs-file") {
+			log.WithFields(log.Fields{
+				"logs-file": *logsFile,
+			}).Fatal("-logs-file is not allowed unless -logs-output is set to 'file'.")
+		}
+		if *logsOutput == "console" {
+			log.SetOutput(os.Stdout)
+		} else {
+			log.WithFields(log.Fields{
+				"logs-output": *logsOutput,
+			}).Fatal("Unknown logs output type")
+		}
+	}
```

Comment:
branch it out into 3 parts is probably easier to reason about: 
```
if logs to file 
    set file 
else if logs to consle 
    set console
else
    fatal error
```

Reply:
The part is totally replaced by a new implementation for support multiple `logs-output`.

Reply:
thanks